### PR TITLE
[TTAHUB-1557] - Header alignment and responsive behavior

### PR DESCRIPTION
**Pull request by Spanglemonkey**
## Description of change

Adjusted css classes to align header content, and add responsive behavior. Also added css changes to support responsive behavior.

## How to test
Look at header on different pages and resize window or use device emulator to see responsive behavior (header is shorter, logo + logo text + avatar is smaller)

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1557

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [x] Code is meaningfully tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [x] UI review complete

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: frontend/src/App.scss [hunk 1]
```diff
@@ -36,9 +36,22 @@ a:visited {
   background-color: $ttahub-blue;
 }
 
+smart-hub-title {
```
Comment (thewatermethod) [inline frontend/src/App.scss:39]: missing the dot, so I don't think this is applying to anything

maybe we can lose the entire rule if that's the case

Comment (Spanglemonkey) [reply frontend/src/App.scss:39]: Nope - need the dot. What is the process for fixing it?

**Review by thewatermethod — APPROVED** [review]

File: frontend/src/App.scss (hunk 1 above)
Comment (Spanglemonkey) [reply frontend/src/App.scss]: the 1rem declaration was the mobile size, it is larger at larger widths.

Comment (thewatermethod) [reply frontend/src/App.scss]: Yep. When I went down to mobile size, it looks like the default font size for views smaller than 64rem is 1rem.

**Pull request merged**